### PR TITLE
Decoupled frontend from firebase

### PR DESCRIPTION
**Pull request by AtriLahiji**
I removed all references to Firebase from the frontend. Moved the functions to helper functions in the new "backend" directory. How does it look @IxLahiji ?

### CODE REVIEW

File: components/dialog/EditPollDialog.vue [hunk 1]
```diff
@@ -231,9 +231,11 @@ along with Pollo.  If not, see <https://www.gnu.org/licenses/>. -->
 </template>
 
 <script>
-import firebase from '@firebase/app'
+import { mapGetters } from 'vuex'
```
Comment (IxLahiji) [inline components/dialog/EditPollDialog.vue:234]: Why is this in the generalization PR?

Comment (AtriLahiji) [reply components/dialog/EditPollDialog.vue:234]: Needed for this:
`...mapGetters([
      'GET_LOGGED_IN_USER'
    ]),`

File: package.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
     "file-saver": "^2.0.2",
     "json2csv": "^4.5.4",
     "nuxt": "^2.12.1",
-    "vue-chartjs": "^3.5.0",
+    "vue-graph": "^0.8.7",
```
Comment (IxLahiji) [inline package.json:29]: Why change the graph library?

Comment (AtriLahiji) [reply package.json:29]: vue-chartjs struggled to update correctly when I moved the watchers to their own file. In my testing vue-graph worked better.

**Pull request merged**